src/index.js

```diff
@@ -2,23 +2,28 @@ import React from "react";
 import ReactDOM from "react-dom";
```

Comment:
try to order the imports according to the level of their dependencies, for example, first core imports then components, then services, then store, group them in a way you will maintain